internal/plugin/connectors/tcp/proxy_service.go

```diff
@@ -6,6 +6,7 @@ import (
 	"net"
 
 	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/pkg/errors"
```

Comment:
The changes in this file really feel like a separate PR, but that's ok... At the very least we should clean the history so that these changes are there own commit with a clear message explaining why we're making them and what benefits the wrapping provides us.